src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -1221,6 +1272,42 @@ public ConnectionStringReference(IResourceWithConnectionString resource, bool op
         System.Threading.Tasks.ValueTask<string?> IValueProvider.GetValueAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
     }
 
+    [System.Diagnostics.CodeAnalysis.Experimental("ASPIRECOMPUTE001", UrlFormat = "https://aka.ms/aspire/diagnostics/{0}")]
```

Comment:
@captainsafia - I don't think these should reuse `ASPIRECOMPUTE001`. As that was already taken:

https://aspire.dev/diagnostics/aspirecompute001/